### ISSUE TITLE
Downgrading packages to more stable versions fails in non-obvious way if there is no matching package

### ISSUE THREAD

**Issue by Seldaek** [composer/composer#2061]
```
    - Installation request for doctrine/doctrine-fixtures-bundle ~2.1 -> satisfiable by doctrine/doctrine-fixtures-bundle[2.1.x-dev].
    - Removal request for doctrine/doctrine-fixtures-bundle == 2.1.9999999.9999999-dev
```

The error reporting should be adjusted probably if the only matching package is in the installed repo and does not pass a Pool::isAcceptable check

**Comment by Seldaek** [comment]
Needs to be checked for 2.0 with the new pool changes.

**Comment by Seldaek** [comment]
Fixed in 2.0
